UI/Source/CollectionViews/CollectionViewModelDataSource.swift

```diff
@@ -45,22 +45,32 @@ public final class CurrentCollection: ModelCollection, ProxyingCollectionEventOb
 
     public let collectionId: ModelCollectionId
 
-    public private(set) var state: ModelCollectionState = .notLoaded {
-        didSet {
-            observers.notify(.didChangeState(state))
-        }
+    public var state: ModelCollectionState {
+        // `sectionedState` is the source-of-truth for this class.
```

Comment:
maybe log.warn here?

Reply:
oh, sorry. you can definitely call this — the comment was meant to explain _why_ this is a dynamic getter. I'll update it to clarify.